draft-ietf-cbor-7049bis.md

```diff
@@ -37,15 +37,15 @@ normative:
     target: http://www.ecma-international.org/publications/files/ecma-st/ECMA-262.pdf
     title: ECMAScript Language Specification 5.1 Edition
     author:
-    - org: European Computer Manufacturers Association
+      - org: European Computer Manufacturers Association
```

Comment:
markdownlint recommends YAML indentation that makes clear this list is subordinate to the author key.